### PR TITLE
Run as a service

### PR DESCRIPTION
**Pull request by matthiassb**
Includes:
* Commander
* Always-tail

image

### CODE REVIEW

File: bin/raneto.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env node
+
+//Hide Keymetrics banner on first time PM2 is required
+var consoleLog = console.log;
+console.log = function() {};
+var pm2 = require('pm2');
```
Comment (ryanlelek) [inline bin/raneto.js]: PM2 required, please add to package.json

File: package.json [hunk 2]
```diff
@@ -21,6 +21,9 @@
       "url": "https://raw.githubusercontent.com/gilbitron/Raneto/master/LICENSE"
     }
   ],
+  "bin": {
+    "raneto": "./bin/raneto.js"
+  },
```
Comment (ryanlelek) [inline package.json]: I wouldn't deny a request for this one thing, but while you're editing anyway, change this to:
"bin": "./bin/raneto" as per NPM spec (defaults to name of package)

**Comment by matthiassb** [comment]
Addressed concerns

**Comment by ryanlelek** [comment]
Excellent. Confirmed working, merged.
Thank you for your contribution

**Comment by gilbitron** [comment]
I like this. Thanks for the contrib.

**Comment by gilbitron** [comment]
Just noticed the README needs updated to explain how to run as a service. @ryanlelek

**Comment by ryanlelek** [comment]
@gilbitron First draft done.
Please publish v0.8.0 to NPM as well, I don't have access

**Comment by gilbitron** [comment]
@ryanlelek :+1: done.

**Pull request merged**